module/applications/item/item-sheet.mjs

```diff
@@ -428,6 +437,11 @@ export default class ItemSheet5e extends ItemSheet {
     const damage = formData.system?.damage;
     if ( damage ) damage.parts = Object.values(damage?.parts || {}).map(d => [d[0] || "", d[1] || ""]);
 
+    // Handle properties
+    if ( foundry.utils.hasProperty(formData, "system.properties") ) {
+      formData.system.properties = filteredKeys(formData.system.properties);
```

Comment:
We can preserve any currently unconfigured properties fairly easily I think. It's not clear whether this presents an opportunity to clean up old data, or whether we should preserve it, but I think we should err on the side of preserving it for now until it becomes clear that it would be better to do otherwise.

```suggestion
      const keys = new Set(Object.keys(formData.system.properties));
      const preserve = this.object.system.properties.difference(keys);
      formData.system.properties = [...filteredKeys(formData.system.properties), ...preserve];
```